lib/integration/crud.ts

```diff
@@ -259,6 +260,7 @@ const _onlyIncludePublicProperties = async ({
       return {
         formID: template.formID,
         publishingStatus: template.formConfig.publishingStatus,
+        securityAttribute: template.formConfig?.securityAttribute ?? "",
```

Comment:
The `?` here is not required, since `formConfig` will always be defined in this case.

Reply:
same as [here](https://github.com/cds-snc/platform-forms-client/pull/728#discussion_r856516608)

Reply:
same as [here](https://github.com/cds-snc/platform-forms-client/pull/728#discussion_r856516608)